src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -60,7 +65,7 @@ public PersonBuilder withName(String name) {
     /**
      * Parses the {@code tags} into a {@code Set<Tag>} and set it to the {@code Person} that we are building.
      */
-    public PersonBuilder withTags(String ... tags) {
+    public PersonBuilder withTags(String... tags) {
```

Comment:
Perhaps we could add back the space here String ...

Reply:
Interesting, changing this does not affect checkstyle